templates/taxi/car_detail.html

```diff
@@ -1,8 +1,17 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <ul class="float-right btn-group">
+    <a class="btn btn-secondary" href="update">Edit</a>
```

Comment:
The 'Edit' link should use the `{% url %}` template tag to ensure it points to the correct URL. For example: `{% url 'car-update' car.id %}`.